maestro-ios-xctest-runner/maestro-driver-iosUITests/Routes/Helpers/ScreenSizeHelper.swift

```diff
@@ -0,0 +1,25 @@
+import XCTest
+
+struct ScreenSizeHelper {
+    static func physicalScreenSize() -> (Float, Float) {
```

Comment:
Would we need to account this for other endpoints like swipe?

Reply:
With the current approach - yes.

I admit I forgot about this. I will do that (later today)

Reply:
Done